Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -1649,4 +1649,68 @@ class FeaturevisorInstanceTests: XCTestCase {
         XCTAssertEqual(variableWithDefaultContext!.value as! String, "red")
         XCTAssertEqual(variableWithExtendedContextByCountry!.value as! String, "orange")
     }
+
+    func testShouldParseNilVariables() {
+
+        // GIVEN
```

Comment:
PascalCase -> "Given" to be consistent

Reply:
In this file all tests have upppercase. Maybe should I update them for consistency?